nflow-engine/src/main/java/com/nitorcreations/nflow/engine/listener/WorkflowExecutorListener.java

```diff
@@ -85,6 +85,28 @@ public ListenerContext(AbstractWorkflowDefinition<?> definition, WorkflowInstanc
   void beforeProcessing(ListenerContext listenerContext);
 
   /**
+   * Processing chain.
```

Comment:
WorkflowExecutorListener.process() JavaDoc seems to be misleading: the returned NextAction can only affect workflows instance stateText and nextActivation?